packages/form-js-viewer/test/schemaWithAllComponents.json

```diff
@@ -0,0 +1,482 @@
+        },
+        "disabled": true,
+        "layout": {
+          "row": "Row_0xpl64i"
+        }
+      },
+      {
+        "values": [
+          {
+            "label": "Option 1",
+            "value": "option_1"
+          },
+          {
+            "label": "Option 2",
+            "value": "option_2"
+          },
+          {
+            "label": "Option 3",
+            "value": "option_3"
+          }
+        ],
+        "label": "I am a select",
+        "type": "select",
+        "id": "Field_1iw4ekg",
+        "key": "select",
+        "description": "I am a select description",
+        "validate": {
+          "required": true
+        },
+        "defaultValue": "option_1",
+        "layout": {
+          "row": "Row_084h2jt",
+          "columns": 6
+        }
+      },
+      {
+        "values": [
+          {
+            "label": "Option 1",
+            "value": "option_1"
+          },
+          {
+            "label": "Option 2",
+            "value": "option_2"
+          },
+          {
+            "label": "Option 3",
+            "value": "option_3"
+          }
+        ],
+        "label": "I am a select",
+        "type": "select",
+        "id": "Field_06g19xn",
+        "key": "disabled_select",
+        "description": "I am a select description",
+        "disabled": true,
+        "defaultValue": "option_2",
+        "layout": {
+          "row": "Row_084h2jt",
+          "columns": 10
+        }
+      },
+      {
+        "values": [
+          {
+            "label": "Option 1",
+            "value": "option_1"
+          },
+          {
+            "label": "Option 2",
+            "value": "option_2"
+          },
+          {
+            "label": "Option 3",
+            "value": "option_3"
+          }
+        ],
+        "label": "I am a searcheable select",
+        "type": "select",
+        "id": "Field_0g90q9s",
+        "key": "searcheable_select",
+        "description": "I am a searcheable select description",
+        "searchable": true,
+        "defaultValue": "option_1",
+        "validate": {
+          "required": true
+        },
+        "layout": {
+          "row": "Row_1cn6eez"
+        }
+      },
+      {
+        "values": [
+          {
+            "label": "Option 1",
+            "value": "option_1"
+          },
+          {
+            "label": "Option 2",
+            "value": "option_2"
+          },
+          {
+            "label": "Option 3",
+            "value": "option_3"
+          }
+        ],
+        "label": "I am a disabled searcheable select",
+        "type": "select",
+        "id": "Field_06wzkll",
+        "key": "disabled_searcheable_select",
+        "description": "I am a disabled searcheable select description",
+        "searchable": true,
+        "defaultValue": "option_2",
+        "disabled": true,
+        "layout": {
+          "row": "Row_0benm2m"
+        }
+      },
+      {
+        "type": "image",
+        "alt": "An image",
```

Comment:
The report shows an error for the image: 
![image](https://user-images.githubusercontent.com/9433996/235886131-e3b3c4ed-d458-45d9-820a-8edf72321cdf.png)

Might there be a difference in the environment, or is it just random? I just want to make sure this does not fail randomly in the future.